rpcs3/main.cpp

```diff
@@ -470,11 +477,32 @@ int main(int argc, char** argv)
 	}
 #endif
 
+	// If we're in headless, or non-gui, or we pass in the command line to exit after running, then we exit the
+	// application when the Emulator terminates (potentially returning success / error return code)
+	bool exit_after = parser.isSet(arg_headless) || parser.isSet(arg_no_gui) || parser.isSet(arg_exit_after);
+
+	if(parser.isSet(arg_install_pkgs) && !parser.value(arg_install_pkgs).isEmpty())
+	{
+		const auto pkg_paths = parser.value(arg_install_pkgs).split(',');
+		for (const auto pkg_path : pkg_paths)
+		{
+			if (!pkg_path.isEmpty())
+			{
+				if (!Emu.InstallPkg(pkg_path.toStdString()) && exit_after)
```

Comment:
This doesn't show the user the installation progress in any way...
Neither in the GUI nor otherwise.

And Emu.InstallPkg is only supposed to be used by the Emulator.
Regular installation is different and handled by the main_window

Reply:
"This doesn't show the user the installation progress in any way...
Neither in the GUI nor otherwise."
This is a common complaint with the entire CLI environment in rpcs3.
Having log errors / notices sent to stderr/stdout would resolve these kind of things... although I think it would spit out too much.

Reply:
This isn't really about the cli.... it's just not correct.
You can't just start a wrong installer and then don't even handle the progress.